### ISSUE TITLE
Device and Configuration autoloader

### ISSUE THREAD

**Issue by omaemaru** [Raptor007/Falcon4toSaitek#2]
Awesome tool, @Raptor007. Could you implement it so that every time the executable is launched, the devices and configuration are automatically picked without having to manually do so?

**Comment by omaemaru** [comment]
in addition, could an automatic launch be implemented as well, without having to press start? So everytime the executable Falcon BMS is detected, it's launched.

**Comment by Raptor007** [comment]
Yeah I should be able to do all of that.  One issue I ran into with multiple MFDs is that there didn't seem to be any consistency in their detection order, but if you use the same profile for all (or only have one) that wouldn't really matter.

I think if you check "Wait for Falcon", uncheck "Automatically Launch Falcon", and then click "Start", it should sort of behave as you describe for automatic launch: it waits until it detects Falcon running, then begins output to the configured devices.  You could leave it minimized in that state.

It's been a while since I've worked on this project, but I think I was playing with the idea of having it able to start up as just a tray icon that's ready to begin output whenever it detects an instance of Falcon running, sort of like the MFD Extractor.  Is that basically what you'd like it to do?

**Comment by omaemaru** [comment]
> Yeah I should be able to do all of that. One issue I ran into with multiple MFDs is that there didn't seem to be any consistency in their detection order, but if you use the same profile for all (or only have one) that wouldn't really matter.
> 
Unfortunately, I do not have MFDs to test this for you, but it would be a good heads-up for others to let them know.

> I think if you check "Wait for Falcon", uncheck "Automatically Launch Falcon", and then click "Start", it should sort of behave as you describe for automatic launch: it waits until it detects Falcon running, then begins output to the configured devices. You could leave it minimized in that state.
> 

Yes, it behaves the way you have written.

> It's been a while since I've worked on this project, but I think I was playing with the idea of having it able to start up as just a tray icon that's ready to begin output whenever it detects an instance of Falcon running, sort of like the MFD Extractor. Is that basically what you'd like it to do?

Correct, my setup is the following: whenever I launch BMS, I have a bat file that opens all the required programs, such as Opentrack, Joystick Gremlin, your software, and so on. I wish to avoid the hassle of selecting the profiles and clicking start and have your program automatically do so by detecting that BMS has already been launched.
